app/submitter/submitter.py

```diff
@@ -7,16 +8,41 @@
 logger = get_logger()
 
 
-class LogSubmitter():  # pylint: disable=no-self-use
+class LogSubmitter:
+    @staticmethod
+    def send_message(message, **kwargs):
```

Comment:
I think `case_id` and `tx_id` should be named args rather than kwargs. They are always passed to `send_message`.

Reply:
Addressed.